integreat_compass/cms/templates/_base.html

```diff
@@ -11,7 +12,11 @@
                      alt="{% translate "Logo" %}"
                      class="h-24 w-48">
             </a>
-            <a href="#" class="ml-6 mr-4 hover:text-gray-900">{% translate "Dashboard" %}</a>
+            <a href="" class="ml-6 mr-4 hover:text-gray-900">{% translate "Dashboard" %}</a>
+            {% if user|has_group:"BOARD_MEMBER" %}
```

Comment:
```suggestion
            {% if perms.cms.add_vote %}
```
Groups are just an abstraction to make it easier to assign the same set of permissions to multiple users, however in the templates & views we want to check for the actual permissions instead of group memberships, so even if we change the permissions of a group in the future, the functionality still stays consistent.